src/main/webapp/WEB-INF/jsp/bdrs/user/record/render/attributeRenderer.jsp

```diff
@@ -24,6 +24,10 @@
 
 <% pageContext.setAttribute("replaceChar", " "); %>
 
+<%-- Access the facade to retrieve the preference information --%>
+<jsp:useBean id="bdrsPluginFacade" scope="request" type="au.com.gaiaresources.bdrs.servlet.BdrsPluginFacade"></jsp:useBean>
```

Comment:
Is there a chance this bean isn't available?  Or is that in itself an error condition (in which case assuming it's available here is fine)?

Just thinking that the next line, which calls a method on that bean, would cause an NPE if it isn't there.